pallets/parachain-staking/src/weights.rs

```diff
@@ -189,37 +189,37 @@ impl WeightInfo for () {
 			.saturating_add(RocksDbWeight::get().writes(3 as Weight))
 	}
 	fn set_collator_commission() -> Weight {
-		(32_000_000 as Weight)
+		(31_000_000 as Weight)
 			.saturating_add(RocksDbWeight::get().reads(5 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(3 as Weight))
 	}
 	fn set_blocks_per_round() -> Weight {
-		(84_000_000 as Weight)
+		(85_000_000 as Weight)
 			.saturating_add(RocksDbWeight::get().reads(6 as Weight))
 			.saturating_add(RocksDbWeight::get().writes(4 as Weight))
 	}
 	fn force_leave_candidates() -> Weight {
-		(96_000_000 as Weight)
-			.saturating_add(RocksDbWeight::get().reads(8 as Weight))
-			.saturating_add(RocksDbWeight::get().writes(6 as Weight))
+		(632_000_000 as Weight)
```

Comment:
This increased once we added the maximum number of collator candidates and also filled up the leaving collator with the maximum number of nominations.